cmd/authenticator_test.go

```diff
@@ -44,8 +44,9 @@ func TestTerminalAuthenticatorWithNoAcctWithInitialPwd(t *testing.T) {
 	}}
 
 	auth.Authenticate(app.Store, "somepassword")
-	assert.Equal(t, true, exited)
-	assert.Equal(t, 1, rval)
```

Comment:
You can get rid of the `rval` variable in this test too, looks like it's not used anymore.

Reply:
Removed. Sorry about slipping with this, my excuse was that it was AM.

Reply:
Haha, no worries. Thanks for fixing it up!